backend/zeno_backend/processing/filtering.py

```diff
@@ -115,6 +115,8 @@ def table_filter(
     filter_result: sql.Composed | None = None
     if filter_predicates is not None and len(filter_predicates.predicates) > 0:
         filter_result = filter_to_sql(filter_predicates, project, model)
+        if filter_result == sql.Composed([]):
+            filter_result = None
```

Comment:
Should we do this in `filter_to_sql`? Don't know if there are more places where this is needed.

Reply:
Yup good point, figured out how to do it in the chart.